tutorials/lodash/difference.md

```diff
@@ -0,0 +1,43 @@
+The `difference()` function in lodash takes two arguments, the array in question and an array of values to compare against, and returns a new array.
```

Comment:
1. Capitalize "Lodash"
2. More concise, and more info on what the function actually does. "in question", etc isn't useful. Replace with "takes two arrays, and returns an array containing all values in the first array that are not in the second array."